homeassistant/components/subaru/remote_service.py

```diff
@@ -0,0 +1,40 @@
+"""Remote vehicle services for Subaru integration."""
+import logging
+
+from subarulink.exceptions import SubaruException
+
+from homeassistant.exceptions import HomeAssistantError
+
+from .const import DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_call_remote_service(hass, controller, cmd, vin, car_name):
+    """Execute subarulink remote command with start/end notification."""
+    hass.components.persistent_notification.create(
+        f"Sending {cmd} command to {car_name}\nThis may take 10-15 seconds",
+        "Subaru",
+        DOMAIN,
+    )
+    _LOGGER.debug("Sending %s command command to %s", cmd, car_name)
+    success = False
+    err_msg = ""
+    try:
+        success = await getattr(controller, cmd)(vin)
+    except SubaruException as err:
+        err_msg = err.message
+
+    hass.components.persistent_notification.dismiss(DOMAIN)
+    if success:
+        hass.components.persistent_notification.create(
+            f"{cmd} command successfully completed for {car_name}",
+            "Subaru",
+        )
+        _LOGGER.debug("%s command successfully completed for %s", cmd, car_name)
+        return True
+
+    hass.components.persistent_notification.create(
+        f"{cmd} command failed for {car_name}: {err_msg}", "Subaru"
+    )
```

Comment:
```suggestion
```

We don't usually show persistent notifications for service calls.  raising HomeAssistantError is good enough

Reply:
Removed